packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -626,6 +597,76 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		}
 	}
 
+	/**
+	 * Creates a {@link RevertibleAlpha} object that can undo a specific change in the tree's history.
+	 * Revision must exist in the given {@link TreeCheckout}'s branch.
+	 *
+	 * @param revision - The revision tag identifying the change to be made revertible.
+	 * @param kind - The kind of commit (e.g., Default, Undo, Redo) this revertible represents.
```

Comment:
a bit inaccurate, it's the kind of commit that produced this revertible i.e.  if the kind is an undo then the commit that is going to be reverted was an undo while the revertible should likely go on the redo stack

Reply:
Yeah, good point. I've rephrased the explanation.